enterprise/frontend/src/embedding-sdk/components/public/InteractiveDashboard/EditableDashboard.tsx

```diff
@@ -0,0 +1,142 @@
+
+const mapStateToProps = (state: State) => {
+  return {
+    canManageSubscriptions: canManageSubscriptions(state),
+    isAdmin: getUserIsAdmin(state),
+    isNavbarOpen: getIsNavbarOpen(state),
+    isEditing: getIsEditing(state),
+    isSharing: getIsSharing(state),
+    dashboardBeforeEditing: getDashboardBeforeEditing(state),
+    isEditingParameter: getIsEditingParameter(state),
+    isDirty: getIsDirty(state),
+    dashboard: getDashboardComplete(state),
+    slowCards: getSlowCards(state),
+    parameterValues: getParameterValues(state),
+    loadingStartTime: getLoadingStartTime(state),
+    clickBehaviorSidebarDashcard: getClickBehaviorSidebarDashcard(state),
+    isAddParameterPopoverOpen: getIsAddParameterPopoverOpen(state),
+    sidebar: getSidebar(state),
+    pageFavicon: getFavicon(state),
+    documentTitle: getDocumentTitle(state),
+    isRunning: getIsDashCardsRunning(state),
+    isLoadingComplete: getIsDashCardsLoadingComplete(state),
+    isHeaderVisible: getIsHeaderVisible(state),
+    isAdditionalInfoVisible: getIsAdditionalInfoVisible(state),
+    selectedTabId: getSelectedTabId(state),
+    isNavigatingBackToDashboard: getIsNavigatingBackToDashboard(state),
+  };
+};
+
+const mapDispatchToProps = {
+  ...dashboardActions,
+  closeNavbar,
+  setErrorPage,
+};
+
+const connector = connect(mapStateToProps, mapDispatchToProps);
+type ReduxProps = ConnectedProps<typeof connector>;
+
+type InteractiveDashboardProps = {
+  dashboardId: DashboardId;
+  initialParameterValues: Query;
+
+  downloadsEnabled?: boolean;
+  navigateToNewCardFromDashboard?: (
+    opts: NavigateToNewCardFromDashboardOpts,
+  ) => void;
+
+  questionHeight?: number;
+  plugins?: SdkPluginsConfig;
+  className?: string;
+} & DashboardFullscreenControls &
+  DashboardRefreshPeriodControls &
+  PublicOrEmbeddedDashboardEventHandlersProps;
+
+const EditableDashboardInner = ({
+  dashboardId,
+  dashboard,
+  initialParameterValues,
+  refreshPeriod,
+  onRefreshPeriodChange,
+  setRefreshElapsedHook,
+  isFullscreen,
+  onFullscreenChange,
+  isEditing,
+  navigateToNewCardFromDashboard,
+  downloadsEnabled,
+  onLoad,
+  onLoadWithoutCards,
+  ...restProps
+}: InteractiveDashboardProps & ReduxProps) => {
+  useDashboardLoadHandlers({ dashboard, onLoad, onLoadWithoutCards });
+
+  return (
+    <Dashboard
+      dashboardId={dashboardId}
+      dashboard={dashboard}
+      parameterQueryParams={initialParameterValues}
+      refreshPeriod={refreshPeriod}
+      onRefreshPeriodChange={onRefreshPeriodChange}
+      setRefreshElapsedHook={setRefreshElapsedHook}
+      isNightMode={false}
+      onNightModeChange={_.noop}
+      hasNightModeToggle={false}
+      isFullscreen={isFullscreen}
+      onFullscreenChange={onFullscreenChange}
+      navigateToNewCardFromDashboard={navigateToNewCardFromDashboard}
+      isEditing={isEditing}
+      downloadsEnabled={downloadsEnabled}
+      onSetErrorPage={console.error}
+      {...restProps}
```

Comment:
nit: we don't need to pass the `dashboardId`, `dashboard`, `refreshPeriod` and other props that has already been passed by `restProps` here